internal/pkg/agent/cmd/enroll_cmd.go

```diff
@@ -507,16 +509,10 @@ func (c *enrollCmd) enrollWithBackoff(ctx context.Context, persistentConfig map[
 		return nil
 	}
 
-	const deadline = 10 * time.Minute
-	const frequency = 60 * time.Second
-
-	c.log.Infof("1st enrollment attempt failed, retrying for %s, every %s enrolling to URL: %s",
-		deadline,
-		frequency,
-		c.client.URI())
+	c.log.Infof("1st enrollment attempt failed, retrying enrolling to URL: %s with exponential backoff (init %s, max %s)", c.client.URI(), enrollBackoffInit, enrollBackoffMax)
 	signal := make(chan struct{})
 	defer close(signal)
-	backExp := backoff.NewExpBackoff(signal, frequency, deadline)
```

Comment:
Naming of these parameters was incorrect, hence the values were most likely incorrect as well.